internal/commands/sbomtest/summary.go

```diff
@@ -0,0 +1,44 @@
+package sbomtest
+
+import (
+	"encoding/json"
+
+	"github.com/snyk/go-application-framework/pkg/local_workflows/json_schemas"
+
+	"github.com/snyk/cli-extension-sbom/internal/snykclient"
+)
+
+type SummaryBuilder struct{}
+
+func (sm SummaryBuilder) BuildTestSummary(resultsSummary *snykclient.SBOMTestSummary) (data []byte, contentType string, err error) {
+	summary := json_schemas.TestSummary{
+		Type: "sbom",
+		Results: []json_schemas.TestSummaryResult{
+			{
+				Severity: "critical",
+				Total:    resultsSummary.VulnerabilitiesBySeverity.Critical,
+				Open:     resultsSummary.VulnerabilitiesBySeverity.Critical,
+			},
+			{
+				Severity: "high",
+				Total:    resultsSummary.VulnerabilitiesBySeverity.High,
+				Open:     resultsSummary.VulnerabilitiesBySeverity.High,
+			},
+			{
+				Severity: "medium",
+				Total:    resultsSummary.VulnerabilitiesBySeverity.Medium,
+				Open:     resultsSummary.VulnerabilitiesBySeverity.Medium,
+			},
+			{
+				Severity: "low",
+				Total:    resultsSummary.VulnerabilitiesBySeverity.Low,
+				Open:     resultsSummary.VulnerabilitiesBySeverity.Low,
+			},
+		},
+	}
+	data, err = json.Marshal(summary)
+	if err != nil {
+		return nil, "", err
+	}
+	return data, "application/json; type=snyk-finding-summary", nil
```

Comment:
should this mime type not be in the framework? hardcoding it here seems a bit fragile, since it's an identifier. i was also under the impression that we would rename it (seen discussions about this).

Reply:
oh yeah, that's part of the `go-application-framework`, missed that.